app/views/admin/index.html.erb

```diff
@@ -21,6 +21,7 @@
             <td><%= request.origin_location %></td>
             <td><%= request.destination %></td>
             <td><%= request.item_id %></td>
+            <td><%= request.item_title %></td>
```

Comment:
* Added SearchWorks link to title
* Added requester info and link

![image](https://cloud.githubusercontent.com/assets/302258/7405453/873b9166-eea7-11e4-910f-de53774224d8.png)